### PR TITLE
refactor: Move calc of slide width and height to Backend

### PR DESCRIPTION
**Pull request by paultrudel**
### What does this PR do?

Removes the `addSlidePositionsPubMsg` that was sent from the html5 client `akka-apps` after a presentation was uploaded to store presentation size data in Postgres. This data is now sent in the `PresentationConversionCompletedSysPubMsg`.

### Motivation

The purpose of this refactor is to send the presentation size data in an already existing message rather than using a new redundant message.

### CODE REVIEW

**Comment by gustavotrott** [comment]
When the meeting was just created and default presentation is set as current, the initial values of Height does not match with what it used to be in Mongo:

image

I guess it is because Meteor inspects the image file to get the height and width:
image

whereas bbb-web/akka-apps are always using the default value as initial Height and Width:
image

**Comment by paultrudel** [comment]
> When the meeting was just created and default presentation is set as current, the initial values of Height does not match with what it used to be in Mongo:
> 
> image
> 
> I guess it is because Meteor inspects the image file to get the height and width: image
> 
> whereas bbb-web/akka-apps are always using the default value as initial Height and Width: image

Yes, that is one of the issues with just relying on `bbb-web` as it does not have the ability to determine the actual height and width of the presentation so it will have to use default values. I am not sure if there is another way to get the actual values without involving the frontend.

**Comment by antobinary** [comment]
`probe` is done on the backend via a library. Perhaps we could check the width/height within bbb-web? On conversion step?

File: bbb-common-web/src/main/scala/org/bigbluebutton/api2/MsgBuilder.scala [hunk 1]
```diff
@@ -65,6 +70,19 @@ object MsgBuilder {
   }
 
   def generatePresentationPage(presId: String, numPages: Int, presBaseUrl: String, page: Int): PresentationPageConvertedVO = {
+    def parseDimension(value: String): Option[Int] = {
```
[the remark this answers is not in the document]
Comment (paultrudel) [reply bbb-common-web/src/main/scala/org/bigbluebutton/api2/MsgBuilder.scala:73]: It is not being used. I forgot to remove it.

**Review by gustavotrott — APPROVED** [review]
LGTM!

**Pull request merged**